src/server/scripts/Northrend/VioletHold/boss_zuramat.cpp

```diff
@@ -179,6 +171,43 @@ class boss_zuramat : public CreatureScript
             if (victim->GetTypeId() == TYPEID_PLAYER)
                 Talk(SAY_SLAY);
         }
+
+        void UpdateAI(uint32 diff) override
+        {
+            //Return since we have no target
+            if (!UpdateVictim())
+                return;
+
+            events.Update(diff);
+
+            if (me->HasUnitState(UNIT_STATE_CASTING))
+                return;
+
+            switch (uint32 eventId = events.ExecuteEvent())
```

Comment:
/home/jackpoz/trinity/sources/src/server/scripts/Northrend/VioletHold/boss_zuramat.cpp: In member function ‘virtual void boss_zuramat::boss_zuramatAI::UpdateAI(uint32)’:
/home/jackpoz/trinity/sources/src/server/scripts/Northrend/VioletHold/boss_zuramat.cpp:183:28: warning: unused variable ‘eventId’ [-Wunused-variable]
             switch (uint32 eventId = events.ExecuteEvent())
                            ^